NewForce-Cohort-5/tabloidfullstack-actual-spring
# Jt commentform

# Description

As a user I should be able to add a new comment to a post. While the user is viewing the post with list of comments and they select "add comment". A form should be presented to input `Subject` and `Content`. The user ID, Post ID, and creation Date should be assigned automatically. The user should then be returned to the list of comments on the post.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. log in as an exisiting user.
2. navigate to the post tab
3. select a post details link by clicking on a name of a post. 
4. Click on the View Comments button to navigate to comment list.
5. Click add Comment to bring up the form and input the information. 
6. You should be directed back to the last after a submit.
7. Also try to cancel out of the form and you should be returned to the list



# Checklist:
